packages/components/src/Airswap/AirswapBuyCVL.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from "react";
+import makeAsyncScriptLoader from "react-async-script";
+import { Button } from "../Button";
+
+const AIRSWAP_URL = "https://cdn.airswap.io/gallery/airswap-trader.js";
+
+export interface BuyCVLProps {
+  buyFromAddress?: string;
+  buyCVLBtnText?: string | JSX.Element;
+  onClick?(index: number): void;
+}
+
+class BuyCVLBase extends React.Component<BuyCVLProps> {
+  public render(): JSX.Element {
+    return <Button onClick={() => this.displayAirswap()}>{this.props.buyCVLBtnText || "Buy CVL"}</Button>;
+  }
+
+  private displayAirswap(): void {
+    // const mainnet = "";
+    const rinkeby = "0x3e39fa983abcd349d95aed608e798817397cf0d1";
+    const buyFromAddress = this.props.buyFromAddress || "";
+
+    // @ts-ignore
+    window.AirSwap.Trader.render(
+      {
+        mode: "buy",
+        env: "sandbox",
+        token: rinkeby,
+        address: buyFromAddress,
+        onComplete: (transactionId: string) => {
+          console.info("Trade complete. Thank you, come again.", transactionId);
```

Comment:
Are these `console` statements placeholders for future messaging to the user when these methods fire?

Reply:
Yes, this functionality is coming soon. We haven't decided what is going to happen yet.